andela/metis-ah
# #159987392 configure eslint

#### What does this PR do?
Configure eslint as the required linter for the project.
#### Description of Task to be completed?
- Install eslint
- Install eslint-airbnb-base
- Initialize eslint with setup `.eslintrc.json`
- Setup editor config to enforce some styling guides
#### How should this be manually tested?
1. Clone the repository
2. Checkout the `ch-configure-eslint-15998732` branch
3. All `*.js` files should have styling errors flagged
#### Any background context you want to provide?
This configuration should help enforce styling guides accross the team.
#### What are the relevant pivotal tracker stories?
#159987392
